internal/appconfig/serde.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/superfly/flyctl/iostreams"
 )
 
+const flytomlHeader = `# fly.toml file generated for %s on %s\n\n", c.AppName, time.Now().Format(time.RFC3339))
```

Comment:
there's interpolation here that got encoded into the raw string on accident

Reply:
nice catch